agent/agent-bootstrap/src/main/java/com/microsoft/applicationinsights/agent/bootstrap/configuration/ConfigurationBuilder.java

```diff
@@ -64,6 +65,18 @@ public static Configuration create(Path agentJarPath) throws IOException {
         preview.roleInstance =
                 overlayWithEnvVar(APPLICATIONINSIGHTS_ROLE_INSTANCE, WEBSITE_INSTANCE_ID, preview.roleInstance);
 
+        JmxMetric threadCountJmxMetric = new JmxMetric();
+        threadCountJmxMetric.objectName = "java.lang:type=Threading";
+        threadCountJmxMetric.attribute = "ThreadCount";
+        threadCountJmxMetric.display = "Current Thread Count";
+        preview.jmxMetrics.add(threadCountJmxMetric);
+
+        JmxMetric classCountJmxMetric = new JmxMetric();
+        classCountJmxMetric.objectName = "java.lang:type=ClassLoading";
+        classCountJmxMetric.attribute = "LoadedClassCount";
+        classCountJmxMetric.display = "Loaded Class Count";
+        preview.jmxMetrics.add(classCountJmxMetric);
+
```

Comment:
what happens if user has configured one of these in their ApplicationInsights.json file?

Reply:
Good catch.  It will get sent twice.  I will remove the duplicate.

Reply:
With further testing, duplicate only happens if the display name is different.  If they're the same, it only sent once.

Reply:
With this change, it will use customer's display name since that is what they prefer.